support/format/index.js

```diff
@@ -26,6 +65,22 @@ export const formatDate = (value) => {
   return parseDate(value)
 }
 
+/**
+ * Mescla uma string com a outra. Ex.: value = "2018-10-107732123312" truncate = "test"   Result = test-10-107732123312
+ * @param {string} value
+ * @param {string} truncate
+ * @returns {*}
+ */
+export const truncateString = (value, truncate) => {
+  if (!(String(value).length >= String(truncate.length))) {
+    return truncate
+  }
+
+  let _substring = String(value).substring(String(truncate).length)
```

Comment:
Por que let e porque o _ no nome (é alguma notação)?

Reply:
Poderia ser um const msm, não sei porque coloquei como let. Sobre o nome, coloquei o _ só pra indicar que é algo interno no método, mais pode ser alterado também, não tem relevância.

Reply:
Se for usar _ seria interessante a gente aplicar isso em todos os lugares, não? o que acha? aplica em todos e se torna um padrão ou tira desse?

Reply:
Então, eu acho massa, mais ter que fazer isso em tudo agora é tenso. Melhor tirar, não?